<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pqFileMenuBuilder</class>
 <widget class="QMenu" name="pqFileMenuBuilder">
  <property name="title">
   <string>&amp;Файл</string>
  </property>
  <widget class="QMenu" name="menuRecentFiles">
   <property name="title">
    <string>Недавние файлы</string>
   </property>
  </widget>
  <action name="actionServerConnect">
   <property name="icon">
    <iconset resource="../../../Components/Resources/pqComponents.qrc">
     <normaloff>:/pqWidgets/Icons/pqConnect24.png</normaloff>:/pqWidgets/Icons/pqConnect24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Подключение...</string>
   </property>
   <property name="statusTip">
    <string>Подключение</string>
   </property>
  </action>
  <action name="actionServerDisconnect">
   <property name="icon">
    <iconset resource="../../../Components/Resources/pqComponents.qrc">
     <normaloff>:/pqWidgets/Icons/pqDisconnect24.png</normaloff>:/pqWidgets/Icons/pqDisconnect24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Разорвать подключение</string>
   </property>
   <property name="statusTip">
    <string>Разорвать подключение</string>
   </property>
  </action>
  <action name="actionFileSaveScreenshot">
   <property name="icon">
    <iconset resource="../../../Components/Resources/pqComponents.qrc">
     <normaloff>:/pqWidgets/Icons/pqCaptureScreenshot24.png</normaloff>:/pqWidgets/Icons/pqCaptureScreenshot24.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить скриншот...</string>
   </property>
  </action>
  <action name="actionFileSaveAnimation">
   <property name="icon">
    <iconset resource="../../../Components/Resources/pqComponents.qrc">
     <normaloff>:/pqWidgets/Icons/pqSaveAnimation24.png</normaloff>:/pqWidgets/Icons/pqSaveAnimation24.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить &amp;анимацию...</string>
   </property>
   <property name="PV_MUST_BE_MASTER_TO_SHOW" stdset="0">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionFileSaveData">
   <property name="icon">
    <iconset resource="../../../Components/Resources/pqComponents.qrc">
     <normaloff>:/pqWidgets/Icons/pqSave24.png</normaloff>:/pqWidgets/Icons/pqSave24.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить данные...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="PV_MUST_BE_MASTER_TO_SHOW" stdset="0">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionFileLoadServerState">
   <property name="text">
    <string>&amp;Загрузить Состояние...</string>
   </property>
   <property name="PV_MUST_BE_MASTER_TO_SHOW" stdset="0">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionFileSaveServerState">
   <property name="text">
    <string>&amp;Сохранить Состояние...</string>
   </property>
  </action>
  <action name="actionFileSaveGeometry">
   <property name="text">
    <string>Сохранить &amp;Геометрию...</string>
   </property>
   <property name="PV_MUST_BE_MASTER_TO_SHOW" stdset="0">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionFileOpen">
   <property name="icon">
    <iconset resource="../../../Components/Resources/pqComponents.qrc">
     <normaloff>:/pqWidgets/Icons/pqOpen24.png</normaloff>:/pqWidgets/Icons/pqOpen24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Открыть...</string>
   </property>
   <property name="statusTip">
    <string>Открыть</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="PV_MUST_BE_MASTER_TO_SHOW" stdset="0">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionFileExit">
   <property name="text">
    <string>Вы&amp;ход</string>
   </property>
   <property name="statusTip">
    <string>Выход</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Экспорт сцены...</string>
   </property>
  </action>
  <action name="actionFileSaveWindowLayout">
   <property name="text">
    <string>Сохранить макет окна...</string>
   </property>
   <property name="toolTip">
    <string>Сохранить макет окна в файл</string>
   </property>
   <property name="statusTip">
    <string>Сохранить макет окна в файл</string>
   </property>
  </action>
  <action name="actionFileLoadWindowLayout">
   <property name="text">
    <string>Загрузить макет окна...</string>
   </property>
   <property name="toolTip">
    <string>Загрузить макет окна из файла</string>
   </property>
   <property name="statusTip">
    <string>Загрузить макет окна из файла</string>
   </property>
  </action>
  <addaction name="actionFileOpen"/>
  <addaction name="menuRecentFiles"/>
  <addaction name="separator"/>
  <addaction name="actionFileLoadServerState"/>
  <addaction name="actionFileSaveServerState"/>
  <addaction name="separator"/>
  <addaction name="actionFileSaveData"/>
  <addaction name="actionFileSaveScreenshot"/>
  <addaction name="actionExport"/>
  <addaction name="separator"/>
  <addaction name="actionFileSaveAnimation"/>
  <addaction name="actionFileSaveGeometry"/>
  <addaction name="separator"/>
  <addaction name="actionFileLoadWindowLayout"/>
  <addaction name="actionFileSaveWindowLayout"/>
  <addaction name="separator"/>
  <addaction name="actionServerConnect"/>
  <addaction name="actionServerDisconnect"/>
  <addaction name="separator"/>
  <addaction name="actionFileExit"/>
 </widget>
 <resources>
  <include location="../../../Components/Resources/pqComponents.qrc"/>
 </resources>
 <connections/>
</ui>
